packages/webapp/src/pages/index.tsx

```diff
@@ -12,6 +12,33 @@ import {
     Text,
 } from "_app";
 
+interface CommunityStatusProps {
+    memberStats: any
+}
+
+const CommunityStatsCard = ({memberStats}: CommunityStatusProps) => {
+    return memberStats && (
+        <Card title="Community Stats" titleSize={1} className="h-full space-y-8 text-center p-8 lg:p-12 lg:py-24 xl:p-16 text-md lg:text-xl">
+            <p>
```

Comment:
I'd recommend removing the `title` prop here and using a `Heading` inside the `Card` component. Then use flexbox inside the card to center everything. Something like:

```javascript
<Card className="h-full flex flex-col justify-center items-center space-y-4 text-center text-lg lg:text-xl">
    <Heading size={2} className="mb-2">
        Community Stats
    </Heading>
    <Link href="/members" className="font-medium">
        {memberStats.active_members} active member
        {memberStats.active_members !== 1 && "s"}
    </Link>
    <Link
        href="/induction/pending-invitations"
        className="font-medium"
    >
        {memberStats.pending_members} pending invitation
        {memberStats.pending_members !== 1 && "s"}
    </Link>
</Card>
```